app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,17 @@
+from abc import ABC
+
+
+from app.players.player import Player
```

Comment:
One blank line between the different groups of imports is enough, consider it in the future when grouping different imports.